### DISCUSSION TITLE
Any chance of a C# interface?

### DISCUSSION THREAD

**Discussion by acrigney** [o3de/o3de#1892] [General]
Unity is the standard for us C# developers, adding C# would really open up your platform?

**Comment by Racso** [comment]
Hi! There's another issue regarding this topic: https://github.com/o3de/o3de/discussions/1888 - Feel free to join the discussion there!

**Comment by ghost** [comment]
for the love of god, no

**Comment by acrigney** [comment]
Very poisonous response, the performance of C# is really only 5% less than
C++ these days, what is your issue, it would open up the market?
Just do a C# wrapper and you are away?
